spinalcordtoolbox/straightening.py

```diff
@@ -478,10 +477,7 @@ def straighten(self):
                 coord_in_planes_curved = centerline.get_in_plans_coordinates(projected_points_curved,
                                                                              nearest_indexes_curved)
 
-                coord_curved2straight = centerline_straight.points[lookup]
-                coord_curved2straight[:, 0:2] += coord_in_planes_curved[:, 0:2]
-                coord_curved2straight[:, 2] += distances_curved
-
+                coord_curved2straight = centerline_straight.get_inverse_plans_coordinates(coord_in_planes_curved, lookup)
```

Comment:
This is maybe out of scope for this PR, but I'm a bit confused by the old code's use of `plans` vs. `planes` in the naming scheme? I'm wondering... is `plans` a typo?

Reply:
Definitely out of scope. But yes, it should be `planes`. Rather than a typo though, I think it's a case of using the French word instead of the English word. (I've noticed a few places in our codebase which have typical francophone-writing-in-english mistakes.)